refapp/apps/ChatBot/src/libs/semantic-kernel/SemanticKernel.ts

```diff
@@ -2,46 +2,89 @@
 
 import { IAsk } from './model/Ask';
 import { IAskResult } from './model/AskResult';
+import {
+    IKeyConfig, SK_HTTP_HEADER_COMPLETION_BACKEND, SK_HTTP_HEADER_COMPLETION_ENDPOINT, SK_HTTP_HEADER_COMPLETION_KEY, SK_HTTP_HEADER_COMPLETION_MODEL, SK_HTTP_HEADER_EMBEDDING_BACKEND, SK_HTTP_HEADER_EMBEDDING_ENDPOINT, SK_HTTP_HEADER_EMBEDDING_KEY, SK_HTTP_HEADER_EMBEDDING_MODEL, SK_HTTP_HEADER_MSGRAPH
+} from './model/KeyConfig';
 
 interface ServiceRequest {
     commandPath: string;
     method?: string;
     body?: unknown;
-    token?: string;
+    keyConfig: IKeyConfig;
 }
 
 export class SemanticKernel {
     // eslint-disable-next-line @typescript-eslint/space-before-function-paren
     constructor(private readonly serviceUrl: string) { }
 
+    private keyConfig: IKeyConfig = {
+        completionConfig: {
+            backend: 0,
+            label: import.meta.env.VITE_REACT_APP_AZURE_OPEN_AI_COMPLETION_DEPLOYMENT,
+            deploymentOrModelId: import.meta.env.VITE_REACT_APP_AZURE_OPEN_AI_COMPLETION_DEPLOYMENT,
+            endpoint: import.meta.env.VITE_REACT_APP_AZURE_OPEN_AI_ENDPOINT,
+            key: import.meta.env.VITE_REACT_APP_AZURE_OPEN_AI_KEY,
+        },
+        embeddingConfig: {
+            backend: 0,
```

Comment:
Since we are using Azure Open AI, shou we use 1 for the backend? Same for above.

Reply:
Azure Open AI should be 0 (this is from the serviceConfig in the samples)
![Screenshot 2023-03-23 at 10 11 00 AM](https://user-images.githubusercontent.com/125500434/228864889-c3a883b3-670d-4223-a67c-aeabedffabca.png)

Reply:
Azure Open AI should be 0, I think it's mislabeled in some places in the repo 
<img width="373" alt="Screenshot 2023-03-30 at 10 14 32 AM" src="https://user-images.githubusercontent.com/125500434/228873481-37468d3d-b192-4103-bbca-968cd39c0ceb.png">

Reply:
I fixed the inconsistent comments in all the KeyConfig type files